apps/engmanagement/prisma/schema.prisma

```diff
@@ -0,0 +1,235 @@
+generator client {
+  provider = "prisma-client-js"
+  previewFeatures = ["referentialIntegrity", "interactiveTransactions"]
+}
+
+datasource db {
+    provider             = "mysql"
+    url                  = env("DATABASE_URL")
+    referentialIntegrity = "prisma"
+}
+
+model Comment {
+  id                       String  @id @default(uuid())
+  userId                   String
+  text                     String
+  context                  Json?
+  updatedAt                DateTime?
+  createdAt                DateTime          @default(now())
+  user                     User              @relation(fields: [userId], references: [id], onDelete: Cascade)
+}
+
+model LessonProgress {
```

Comment:
of the base schema model names, this is my least favorite as it is highly correlated to the concept of a "lesson" in terms of progress. In the case of this product, progress is by chapter (for instance)